packages/terra-date-picker/package.json

```diff
@@ -30,6 +30,8 @@
     "popper.js": "^1.14.3",
     "prop-types": "^15.5.8",
     "react-datepicker": "^0.64.0",
```

Comment:
Can we drop the react-datapicker dep now?

Reply:
https://github.com/cerner/terra-framework/pull/573/commits/c9128e3b0ebe8368a5a63e61c6ba99ff7a79484d